apps/chat/README.md

```diff
@@ -113,14 +113,22 @@ Skip to [Creating a Channel](#creating-a-channel)
 ### Creating a channel
 
 1. Sign in to the client.
-1. In the **Channels** pane, choose the menu (•••).
-1. In the **Add channel** box, enter a channel name. Choose a desired type and mode.
-1. Choose **Add**.
+2. In the **Channels** pane, choose the menu (•••).
+3. In the **Add channel** box, enter a channel name. Choose a desired privacy and mode.
+4. 
+  - **Create Standard channel**
+  a. Select channel Type `Standard`
+
+  - **Create Elastic channel**
+  a. Select channel Type `Elastic` 
+  b. Enter values of `Maximum SubChannels`, `Target Memberships Per SubChannel` and `Scale-In Minimum Memberships`.
+
+5. Choose **Add**.
 
 **Sample code**
 
 ```js
-async function createChannel(appInstanceArn, name, mode, privacy, userId) {
+async function createChannel(appInstanceArn, name, mode, privacy, elasticChannelConfiguration, userId) {
```

Comment:
Any suggestion on what `elasticChannelConfiguration` looks like? would be good to have a example in code here.

Reply:
Addding